src/card.css

```diff
@@ -0,0 +1,95 @@
+html {
+  font-family: Roboto, sans-serif;
+  color: var(--main-text-color);
+  font-weight: 400;
+}
+
+.body-content {
+  width: 200px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 16px 4px;
+}
+
+.card__production-code {
+  margin: 0 16px 16px;
+  color: var(--secondary-text-color);
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__review {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+.card__reviews-text {
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price-container {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 16px;
+}
+
+.price {
```

Comment:
move each bem block to separate file

Reply:
its not seperate bem block, its card block element .card__price-container. I have only two bem blocks acording to requests: card and stars. Both are in seperate files.

Reply:
according to bem naming it is separate bem block

Reply:
so its all about wrong naming?